pages/profile/views/index.jsx

```diff
@@ -66,13 +66,13 @@ class Index extends React.Component {
         </Fragment>
       }
       {
-        model.asruInspector && model.asru.length && <Fragment>
+        model.asruInspector && model.asru && model.asru.length && <Fragment>
           <h3>Inspector for:</h3>
           { this.renderAsruAssociation(model.asru) }
         </Fragment>
       }
       {
-        model.asruLicensing && model.asru.length && <Fragment>
+        model.asruLicensing && model.asru && model.asru.length && <Fragment>
```

Comment:
Where we have (mildly) complex conditions like this that are used in multiple places then it's probably worth abstracting them into `const`s.

Reply:
cool, I will do it on the next PR